backend/ml_model/repository/safe_test.py

```diff
@@ -0,0 +1,36 @@
+from sklearn.model_selection import train_test_split, GridSearchCV
+from sklearn.tree import DecisionTreeClassifier
+
+
```

Comment:
Also this isnt really testing so the naming is confusing

Reply:
Makes sense, I'll change that